Documentation/contributing/development/contributing_guide.rst

```diff
@@ -402,3 +402,17 @@ Cilium follows the real names policy described in the CNCF `DCO Guidelines v1.0
 
     Your real name should not be an anonymous id or false name that
     misrepresents who you are.
+
+.. _contributor_ladder:
+
+Contributor Ladder
+~~~~~~~~~~~~~~~~~~
+
+To help contributors grow in both privileges and responsibilities for the
+project, Cilium also has a `contributor ladder 
```

Comment:
Why “also”?

Reply:
I would say the contributor ladder is a guide to grow in the project, but not the only way. People could become committers without ever becoming reviewers. Not likely, but possible

Reply:
Hmm OK fair enough, although that's not obvious from the text.
I'm not gonna block the PR any longer on this though :+1: